starters/nextjs-starter-approuter-ts/lib/utils.ts

```diff
@@ -80,3 +80,118 @@ export function getSeoMetadata(
     },
   };
 }
+
+function doesCHildContainArticle(
```

Comment:
I think these util functions belong in the SDK; not rely on every adopter to write their own content structure parsing logic.